docs/config.rst

```diff
@@ -140,3 +140,15 @@ Applicable to in-silico digestion
    +----------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
    |     db                     | Defines whether the digestion should contain only targets, only decoys or both (concatenated); can be "target", "decoy" or "concat"; default = "concat"            |
    +----------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Applicable to local prediction and transfer learning
+----------------------------------------------------
+
+.. table::
+   :class: fixed-table local-prediction-config-table
+
+   +----------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Parameter                  |                             Description                                                                                                                            |
+   +============================+====================================================================================================================================================================+
+   | predictIntensityLocally    | Defines whether an off-line model should be used for predicting insensity; can be True or False; default = False                                                   |
```

Comment:
not sure if we need it specifically, maybe it is possible to realize this directly with the model name... But you may need to leave the table in here, since other options like refinement would still need to go in here. However, these may also be model specific arguments which should be part of the models section...